### ISSUE TITLE
XML Issues

### ISSUE THREAD

**Issue by DalekCraft2** [HaxeFlixel/flixel-ui#235]
To preface, this is HaxeFlixel 4.11.0, with Lime 8.0.0 and OpenFL 9.2.0. I'm using the latest release of Flixel UI, not the latest commit.
- The XML reader automatically changes every name to be all-lowercase, which seems unnecessary.
- Creating a `text` tag for an FlxUINumericStepper will not create an FlxUIInputText, but instead a normal FlxUIText. Not adding a `text` tag will allow for the FlxUINumericStepper to be created normally, but will also make it impossible to change things like the width of the text field.
- The `input_text` tag does create an FlxUIInputText, but it appears with no background and white text by default, like a normal FlxUIText.

**Comment by Geokureli** [comment]
to be honest, flixel-ui is a very one of the least maintained parts of flixel. I personally don't use it, and I highly recommend [haxeui-flixel](http://haxeui.org/getting-started/haxeui-flixel/) instead.

**Comment by DalekCraft2** [comment]
Oh. I had no idea that that existed. Is it compatible with flixel in that its components can be added to states and such?

**Comment by Geokureli** [comment]
yes, I'm working on an example for my Newgrounds API, I'll upload it tomorrow morning and share here.

**Comment by Geokureli** [comment]
@DalekCraft2 Here's is my example:
https://github.com/Geokureli/Newgrounds/blob/flixel-ui/test/haxeui-flixel/Source/states/IntroState.hx#L17

classes that use `ComponentBuilder.build` will extend `FlxSpriteGroup`, and can be added to your state, or other groups.

**Comment by DigiEggz** [comment]
I hope this repository won't be abandoned completely! I rely on it and its useful features for my project. I'll continue to try and maintain as long as we can still get PRs through.

**Comment by Geokureli** [comment]
I realize that porting an existing project to haxe-ui is no small feat, but to be absolutely frank, I despise flixel-ui on a fundemental level and most people who do use it tend to use it for one small thing, and I've seen many people in the haxe forum talk about dropping it entirely.

it would make much much much more sense to recreate haxe-ui's components (without the macro xml building) in a new Flixel-UI repo than to try and maintain flixel-ui IMO

**Comment by DigiEggz** [comment]
I use things like FlxUICursor and the FireTongue implementation and I don't know how long/difficult it would be recreate all of this in haxe-ui.

I'll keep an eye on the progress of haxe-ui, but deadlines prevent me from dropping this library at the moment. if there are plans to shutter this project, official references to it such as [RPGInterface](https://haxeflixel.com/demos/RPGInterface/) should be removed to prevent misguidance and confusion.
